client/components/EntryFormRow/index.js

```diff
@@ -1,20 +1,57 @@
 import React, { Component, PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { getCategories } from 'modules/categories';
+import { actions as AppActions } from 'modules/transactions';
 import DataSelector from './DataSelector';
 import styles from './style.scss';
 
+@connect(
+  state => ({
+    categories: getCategories(state)
+  }),
+  (dispatch => ({
+    actions: bindActionCreators(AppActions, dispatch)
```

Comment:
Something to think about:
* binding all actions under one property
* binding just the actions we use

It's fairly obvious when actions are all from one module, but as soon as we need multiple redux modules it becomes difficult to understand where all these functions are coming from.